unitary/examples/tictactoe/tic_tac_toe_test.py

```diff
@@ -142,16 +144,107 @@ def test_rulesets(run_on_hardware):
         board.move("ab", tictactoe.TicTacSquare.O)
 
 
-@pytest.mark.parametrize("run_on_hardware", [False, True])
-def test_print(run_on_hardware):
-    board = tictactoe.TicTacToe(run_on_hardware=run_on_hardware)
+def test_welcome():
+    board = tictactoe.TicTacToe()
+    game = tictactoe.GameInterface(board)
+    output = game.print_welcome()
+
+    assert (output == """
+        Welcome to quantum tic tac toe!
+        Here is the board:
+        
+        a | b | c
+        -----------
+        d | e | f
+        -----------
+        g | h | i
+        """
+    )
+
+def test_help():
+    original_get_move = tictactoe.GameInterface.get_move
+    tictactoe.GameInterface.get_move = lambda _: 'help'
+    test_file = open("test.txt", "w")
```

Comment:
I am not sure this is a great idea to open a file here.   If you are going to do this, you should use the pytest fixture or a function to make a temp file.  However, you should be able to use `io.StringIO()` instead.

See:
https://github.com/quantumlib/unitary/blob/main/unitary/examples/quantum_rpg/battle_test.py#L8

Reply:
Done